packages/ui/src/components/Visualization/Canvas/FormV2/fields/ObjectField/GroupFields.tsx

```diff
@@ -26,7 +26,11 @@ export const GroupFields: FunctionComponent<GroupFieldsProps> = ({ propName, gro
             requiredProperties.includes(propName),
           );
           const hasObjectTypeProperty = Object.keys(groupProperties).some((propName) => {
-            if (groupProperties[propName].type === 'object' || '$ref' in groupProperties[propName]) {
+            if (
```

Comment:
This was the second part that needed to be fixed to incorporate Array's children